src/components/Header.js

```diff
@@ -1,19 +1,39 @@
 import Link from 'next/link';
-import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+const NaviHeader = styled.header`
+	display: flex;
+	padding: 15px;
+	background-color: #f01878;
+	color: white;
+
+	& a {
+		padding: 0 100px;
+		color: white;
+		text-decoration: none;
+	}
+	& a:hover {
+		color: blanck;
```

Comment:
I guess this should mean 'black'? 🤔 Please, keep in mind that the hover effect won't be visible on a mobile device.
```suggestion
		color: black;
```

Reply:
- Ok, erledigt :-)